morango/sync/utils.py

```diff
@@ -104,30 +102,24 @@ def validate_and_create_buffer_data(  # noqa: C901
         # ensure the partition is within the transfer session's filter
         if not transfer_session.get_filter().contains_partition(record["partition"]):
             raise ValidationError(
-                {
-                    "partition": "Partition {} is not contained within filter for TransferSession ({})".format(
-                        record["partition"], transfer_session.filter
-                    )
-                }
+                "Partition {} is not contained within filter for TransferSession ({})".format(
+                    record["partition"], transfer_session.filter
+                )
```

Comment:
This didn't click for me when you were showing the Exception format the other day, but I remember now that this method of passing a dict to a `ValidationError` is/was a Django thing, where it tells the frontend which field/attribute of the input data the error is associated with (e.g. for rendering forms, so it knows which input box to position the red error message next to). Is that no longer the case?

(Mostly a curiosity, since these aren't models that we'd be editing manually via forms anyway, so this change is good.)

Reply:
Since `validate_and_create_buffer_data` is now only ever called in an operation, the dictionary format no longer serves a purpose.

Reply:
Makes sense -- and I guess ValidationError still makes sense, since what it's doing is... validating :)